src/App.js

```diff
@@ -40,7 +40,9 @@ const withDefault = component => () => (
 
 class App extends Component {
   componentDidMount() {
-    // history.replace('/');
+    // Allow Drupal redirects to determine the initial path.
+    const search = history.location.search.replace('?q=', '');
```

Comment:
Do we really want to change the actual location (as in make another redirect)?

I also tried to navigate to /vfancy/admin/people/permissions path and it doesn't seem to be working.
![screen shot 2018-03-30 at 23 40 40](https://user-images.githubusercontent.com/1845495/38145505-dcc1fab6-3473-11e8-985e-715decd3c4d9.png)

Reply:
> I also tried to navigate to /vfancy/admin/people/permissions path and it doesn't seem to be working.

Reply:
I'm not sure what to do. We could actually remove the original routes from the router. This is really just an implementation detail, which we might come up with a better one in the future.